tests/services/test_planning_svc.py

```diff
@@ -57,7 +57,7 @@ def __init__(self, **kwargs):
 
 
 @pytest.fixture
-def setup_planning_test(loop, ability, agent, operation, data_svc, init_base_world):
+def setup_planning_test(loop, ability, agent, operation, data_svc, event_svc, init_base_world):
```

Comment:
After making the operation state setter fire an event, the planning service tests started failing due to the event service not being instantiated/registered. I added this fixture to initialize the event service and make it discoverable.